pkg/io/ports.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2012-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build linux,amd64 linux,386
+
+package io
+
+import (
+	"fmt"
+)
+
+const portPath = "/dev/port"
+
+// In reads data from the x86 port at address addr. Data must be Uint8, Uint16,
+// Uint32, but not Uint64.
```

Comment:
Is it because two LDRs generated for one read of uint64 bit value? May be you should state it here in comment.

Reply:
No, it's the x86 out command which only accepts DX (16 bits) as an address. See: https://c9x.me/x86/html/file_module_x86_id_222.html